openy.profile

```diff
@@ -269,6 +269,30 @@ function openy_discover_broken_paragraphs(array &$install_state) {
       $query->execute();
     }
   }
+
+  $schedule_paragraph_tables = ['paragraph__field_prgf_schedules_ref', 'paragraph_revision__field_prgf_schedules_ref'];
```

Comment:
That's correct. However this is code duplication. Can we move whole foreach to separate function that will accept:
- Condition field name, e.g. `field_prgf_schedules_ref_plugin_id`
- Serialization field name, e.g. `field_prgf_schedules_ref_plugin_configuration`
- any other params that are different...
- Array with tables.